pyproject.toml

```diff
@@ -38,52 +38,38 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-fastapi = ">=0.95.1"
-pydantic = ">=1"
-sqlalchemy = ">=1.3"
-structlog = ">=20"
-deprecated = ">=1.2"
+python = ">=3.9,<3.13"
+fastapi = ">=0.95.1,<0.112"
```

Comment:
As discussed in similar PRs. This is required given that semantic versioning lets fastapi introduce breaking changes until 1.0. It has the drawback that we have to extend the range before any user can update.